README.md

```diff
@@ -588,7 +588,7 @@ Other Style Guides
     }
 
     // good
-    const foo = function bar() {
+    const foo = function foo () {
```

Comment:
Got it. So I'm just really curious, how does having the names be different make things easier to maintain?